docs/API/NuGet-Protocols.md

```diff
@@ -13,77 +13,89 @@ ms.assetid: ba1d9742-9f1c-42ff-8c30-8e953e23c501
 
 # optional metadata
 
-description: The evolving nuget.org protocols to interact with nuget.org by NuGet clients.
+description: The evolving NuGet.org protocols to interact with NuGet clients.
```

Comment:
The convention in the docs is to always use "nuget.org" in all lowercase, but "NuGet" for the name of the technology. So change this to lowercase.

Reply:
I fixed this in many other places as well.